articles/quickstart/native/ios-swift-siwa/_includes/_login_centralized.md

```diff
@@ -0,0 +1,306 @@
+<!-- markdownlint-disable MD002 MD041 -->
+
+<%= include('../../_includes/_getting_started', { library: 'Swift' }) %>
+
+Add your credentials in the `Auth0.plist` file. If the file does not exist in your project yet, create it:
+
+```xml
+<!-- Auth0.plist -->
+
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+  <key>ClientId</key>
+  <string>${account.clientId}</string>
+  <key>Domain</key>
+  <string>${account.namespace}</string>
+</dict>
+</plist>
+```
+
+<%= include('../../_includes/_ios_dependency_centralized') %>
+
+## Add the "Sign In With Apple" Button
+
+As an example of adding the sign-in button to your view, add a `UIStackView` to your main storyboard and create an outlet in `ViewController.swift`:
+
+```swift
+ @IBOutlet weak var loginProviderStackView: UIStackView!
+```
+
+Next, add the `AuthenticationServices` Framework to your `ViewController.swift`:
+
+```swift
+import AuthenticationServices
+```
+
+Add a function that attaches a `ASAuthorizationAppleIDButton` button to the stack view:
```

Comment:
Add a function that attaches an `ASAuthorizationAppleIDButton` button to the stack view: